src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -235,17 +235,68 @@ const myFade = {
 <header transition:animate={myFade}> ... </header>
 ```
 
-## Preventing client-side navigation
+## Router control
 
-Once you have enabled client-side routing on a page by adding the `<ViewTransitions />` component, every anchor on the page that links to another page on your site will be navigated via client-side routing. There are some cases where you might want to not navigate via CSR. One example is if the link is to any non-page, such as a PDF within your `public/` folder. Or an API route that produces an image.
+The `<ViewTransitions />` router handles navigation by listening to:
 
-In these cases, you can opt out of client-side routing on a per-link basis using the `data-astro-reload` attribute like so:
+- Clicks on `<a>` elements.
+- Backwards and forwards navigation events.
 
-```html
-<a href="/quarterly-earnings.pdf" data-astro-reload>
+The following sections discuss how you can further control when navigation occurs within the router.
+
+### Preventing client-side navigation
+
+There are some cases where you cannot navigate via CSR, since both pages involved must use the `<ViewTransitions />` router to prevent a full-page reload. You may also not want client-side routing on every navigation change, and would prefer a traditional page navigation on select routes instead.
```

Comment:
I believe both the commas here can be removed
```suggestion
There are some cases where you cannot navigate via CSR since both pages involved must use the `<ViewTransitions />` router to prevent a full-page reload. You may also not want client-side routing on every navigation change and would prefer a traditional page navigation on select routes instead.
```